src/connection-manager/connections.ts

```diff
@@ -28,6 +28,8 @@ const networkFee: Map<string, FeeVote> = new Map()
 const CM_INTERVAL = 60 * 60 * 1000
 const WS_TIMEOUT = 10000
 const REPORTING_INTERVAL = 15 * 60 * 1000
+const CLOSING_CODES = [1005, 1006, 1008]
```

Comment:
Please add a comment similar to what you included in the PR description for what these closing codes mean / why they mean we should reconnect (as opposed to just accepting that it should be closed)